src/styles/global.css

```diff
@@ -44,15 +125,23 @@ html {
   --max-content-width-large: 1400px;
   --max-content-width-medium: 1200px;
   --max-content-width-small: 1000px;
+
+  /* Radius */
+  --radius-small: 6px;
+  --radius-medium: 12px;
+  --radius-large: 24px;
+
+  font-size: clamp(1rem, 0.9rem + 0.5vw, 1.2rem);
 }
 
 body {
   margin: 0;
   padding: 0;
-  width: 100vw;
```

Comment:
Was the width removed on purpose? If so, please ignore comment :)

Reply:
Yes I did 😄